src/compas_ui/rhino/objects/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""
+********************************************************************************
+objects
+********************************************************************************
+
+.. currentmodule:: compas_rhino.objects
+
+Classes
+=======
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    MeshObject
+    NetworkObject
+    VolMeshObject
+
+"""
+from __future__ import absolute_import
+
+from ._select import (  # noqa : F401 F403
+    mesh_select_vertex,
+    mesh_select_vertices,
+    mesh_select_face,
+    mesh_select_faces,
+    mesh_select_edge,
+    mesh_select_edges,
+    network_select_node,
+    network_select_nodes,
+    network_select_edge,
+    network_select_edges
+)
+from ._modify import (  # noqa : F401 F403
+    network_update_attributes,
+    network_update_node_attributes,
+    network_update_edge_attributes,
+    network_move_node,
+    mesh_update_attributes,
+    mesh_update_vertex_attributes,
+    mesh_update_face_attributes,
+    mesh_update_edge_attributes,
+    mesh_move_vertex,
+    mesh_move_vertices,
+    mesh_move_face
+)
+
+from ._object import BaseObject
+from .meshobject import MeshObject
+from .networkobject import NetworkObject
+from .volmeshobject import VolMeshObject
+
+from compas.datastructures import Mesh
+from compas.datastructures import Network
+from compas.datastructures import VolMesh
+
+BaseObject.register(Mesh, MeshObject, context='Rhino')
+BaseObject.register(Network, NetworkObject, context='Rhino')
+BaseObject.register(VolMesh, VolMeshObject, context='Rhino')
```

Comment:
this needs to be handled through a plugin

```python
@plugin(category='ui', requires=['Rhino'])
def register_objects():
    RhinoObject.register(Mesh, MeshObject, context='Rhino')
```

otherwise the registration only happens if you import the artists explicitly...

Reply:
there should be an equivalent pluggable in `object.py`

```python
@pluggable(category='ui', selector='collect_all')
def register_objects():
    raise NotImplementedError
```